spec/plans/provision.fmf

```diff
@@ -36,6 +36,22 @@ example: |
         testcloud (libvirt). Testcloud takes care of downloading
         an image and making necessary changes to it for optimal
         experience (such as disabling UseDNS and GSSAPI for SSH).
+
+        image
+            qcov image name or url
```

Comment:
It can be:
- short name (translated to the centos/fedora) by testcloud, e.g. fedora-rawhide (the plan is to add internal plugin for rhel-nightly etc)
- url to the .qcow2 file OR vagrant .box file
- absolute local path to the qcow2

Not sure how to write it in short but correct way.

Reply:
TBH, I’d suggest omitting these changes from this PR completely. Examples, yes, absolutely, but the rest, documenting various keys, especially those that are not related to `become`, would become obsolete Soon(TM) when https://github.com/teemtee/tmt/pull/2549 gets merged. Making changes here just adds more stuff that would need to be moved into actual key documentaion in source code…

Reply:
Removed, per coming PR https://github.com/teemtee/tmt/pull/2549

I added examples instead.